docs/guide/essentials/history-mode.md

```diff
@@ -70,6 +70,29 @@ http.createServer((req, res) => {
 
 For Node.js/Express, consider using [connect-history-api-fallback middleware](https://github.com/bripkens/connect-history-api-fallback).
 
+##### Example
+```bash
+mkdir express_project
+cd express_project
+npm init
+npm i express connect-history-api-fallback -s
```

Comment:
can you remove the `-s` and use `install` instead, please

Reply:
Yes, I fixed.
```
npm install express connect-history-api-fallback
```